src/modules/TeamsList/components/Teams/components/TeamUserTable/TeamUserTable.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { FC } from 'react';
+import { StyledTeamUserTable } from './styled';
+import { User } from 'types';
+import { ExpelButton } from './components/ExpelButton/ExpelButton';
```

Comment:
name file index.tsx and change import to import { ExpelButton } from './components/ExpelButton';

Reply:
done